### PR TITLE
[TASK] Restore the page about login

### PR DESCRIPTION
**Pull request by linawolf**
It is part of the sylabus for the Editors certification

### CODE REVIEW

File: Documentation/Login/Index.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.. include:: ../Includes.txt
+
+.. _login:
+
+=====
+Login
+=====
+
+In TYPO3 CMS an editor's work is done in the backend. So point your browser
```
Comment (brotkrueml) [inline Documentation/Login/Index.rst:9]: "So point your browser ..." is duplicate to line 17-19. I suggest to remove this sentence here.

**Comment by linawolf** [comment]
@brotkrueml done, please have another look

File: Documentation/Login/Index.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+.. important::
+
+   Make sure JavaScript and cookies are enabled in your browser as they are
+   required by the TYPO3 Backend.
+
+Enter your domain name into the address bar of your internet browser and
+append "/typo3" to it to get to the "log in" page of your site.
+For example: :samp:`http://www.example.org/typo3`.
+
+.. figure:: ../Images/BackendLoginPage.png
```
Comment (brotkrueml) [inline Documentation/Login/Index.rst:20]: Sorry, just found now: The image path should be `/Images/AutomaticScreenshots/Login/BackendLoginPage.png`

**Review by brotkrueml — CHANGES_REQUESTED** [review]

File: Documentation/Login/Index.rst (hunk 2 above)
Comment (linawolf) [inline Documentation/Login/Index.rst:20]:
```suggestion
.. figure:: /Images/AutomaticScreenshots/Login/BackendLoginPage.png
```

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**